base/testfiles-ltcmd/ltcmd001.tlg

```diff
@@ -17,97 +17,73 @@ For immediate help type H <return>.
 l. ...}
 You have used \RenewDocumentCommand with a command that was never defined.
 LaTeX will ignore this entire definition.
-> \foo=\protected macro:->\__cmd_start_expandable:nNNNNn {}\foo  \foo  \foo
-code ?{}.
+> \foo=\protected macro:->\__cmd_start_optimised: First definition.
 <recently read> }
 l. ...}
-> \foo code=\protected\long macro:->First definition.
+> \foo code=undefined.
```

Comment:
Polishing: perhaps the setting in `ltcmd001.lvt` can check if `\cs{foo code}` exists before showing it.
https://github.com/latex3/latex2e/blob/0f40cbb8ea0c43592db9d1c96dbb7575166b4a7b/base/testfiles-ltcmd/ltcmd001.lvt#L16-L19
```tex
% module also renamed from "xparse" to "ltcmd"
\cs_new:Npn \ltcmd_show:N #1 {
  \cs_show:N #1
  \cs_if_exist:cT { \cs_to_str:N #1 ~code }
    { \cs_show:c { \cs_to_str:N #1 ~code } }
}
```

Reply:
I thought about that, but isn't the entire point here that most `ltcmd` stuff does have this, so we should be explicit when it's not present?

Reply:
I'm OK to both ways.

Reply:
Latest revision is back using this internal ...